pandas/core/arrays/period.py

```diff
@@ -372,15 +372,13 @@ def __setitem__(
             value = period_array(value)
 
             if self.freqstr != value.freqstr:
```

Comment:
`PeriodArray.__setitem__` is being removed, so I was hoping to not change these.